xconn/_client/helpers.py

```diff
@@ -474,3 +477,22 @@ def assemble_call_details(uri: str, meta: ProcedureMetadata, invocation: Invocat
         details[meta.call_details_field] = CallDetails(invocation.details)
 
     return details
+
+
+def load_config_from_env() -> ClientConfig:
+    url = os.environ.get("XCONN_URL", None)
```

Comment:
```suggestion
    url = os.environ.get("XAPP_URL", None)
```

need to change that in all places